assets/css/homepage.scss

```diff
@@ -0,0 +1,233 @@
+---
+---
+
+// Imports -------------- //
+
+@import 'lib/bourbon/bourbon';
+@import 'lib/neat/neat';
+@import 'all';
+
+// Variables ------------ //
+
+$text-width:      660px;
+$section-padding: 7.2rem;
+$font-lead:   2rem;
+
+// Styles --------------- //
+
+p {
+  font-family: $font-sans;
+  font-size: 1.7rem;
+}
+
+.skipnav {
+  padding: 10px 15px;
+  position: absolute;
+  top: -42px;
+  left: 0px;
+  color: $color-base;
+  background: transparent;
+  @include transition(top 1s ease-out, background 1s linear);
+  z-index: 100;
+  &:focus {
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    background: white;
+    outline: 0;
+    @include transition(top .1s ease-in, background .5s linear);
+  }
+}
+
+.usa-disclaimer {
+  background-color: $color-gray-lighter;
+  font-size: 1.5rem;
+  padding: {
+    bottom: .5rem;
+    top: .5rem;
+  }
+}
+
+.usa-disclaimer-stage {
+  float: right;
+}
+
+.usa-intro {
```

Comment:
Don't we already have classes to do stuff like this? Like the grid system or the usa-content?